### PR TITLE
Removed steps to recover deleted items in change-deleted-item-retention.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2736.

@v-smandalika this article is related to change how long the deleted items are retained. To recover deleted items, there is another article, and the link is at the bottom of this one.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ manager: serdars
 If you've *permanently* deleted an item in Microsoft Outlook or Outlook on the web (formerly known as Outlook Web App), the item is moved to a folder (**Recoverable Items** \> **Deletions**) and kept there for 14 days, by default. You can change how long items are kept, up to a maximum of 30 days.
 
 > [!NOTE]
-> You must use the new Exchange admin center (EAC) or Exchange Online PowerShell to make the change. Unfortunately, you can't currently do this directly in the Outlook or Outlook on the web.
+> You must use Exchange Online PowerShell to make the change. Unfortunately, you can't currently do this directly in the Outlook or Outlook on the web.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md:29]:
```suggestion
> You must use Exchange Online PowerShell to make the change. Unfortunately, you can't currently do this directly in Outlook or Outlook on the web.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md (hunk 1 above)
Comment (dariomws) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md:29]: Are you sure about this? I think I've never seen "the Outlook" as it's a product name.

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md:29]: Yes, that is correct, it should not be "the Outlook". That is why I've suggested removing the word "the".

Comment (dariomws) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md:29]: Oh, sorry, I mixed the comments. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention.md]: No worries. I thought so.

**Pull request merged**